main/codes/tic_tac_toe/logical/TicTacToeLogicalOperator.py

```diff
@@ -35,14 +36,29 @@ def __init__(
                 which operator this represents - either PauliLetter('X') or PauliLetter('Z')
             logical_qubit:
                 the logical qubit this operator is part of.
+            gauge_factor:
+                The gauge factor of the code this operator is part of.
         """
         assert logical_letter in [PauliLetter('X'), PauliLetter('Z')]
 
-        self._at_round = {-1: initial_paulis}
-        self.last_updated = -1
-        self.logical_letter = logical_letter
         self.is_vertical = is_vertical
+        self.logical_letter = logical_letter
         self.logical_qubit = logical_qubit
+
+        # This is a hacky way to include the gauge factor stuff.
+        # pj todo temp fix
+        if gauge_factors == 1:
```

Comment:
Why default to 1? Feels like default here should be 'None'?

Reply:
Gauge_factor 1 means each measurement is repeated once

Reply:
I have clarified this in the docstring of the gauge codes